perceval/simulators/simulator.py

```diff
@@ -78,13 +78,23 @@ def precision(self, value: float):
     def set_precision(self, precision: float):
         self.precision = precision
 
+    # TODO: remove for PCVL-786
+    @deprecated(version="0.11.1", reason="Use set_min_detected_photons_filter instead")
     def set_min_detected_photon_filter(self, value: int):
         """
         Set a minimum number of detected photons in the output distribution
 
         :param value: The minimum photon count
         """
-        self._min_detected_photons = value
+        self._min_detected_photons_filter = value
+
+    def set_min_detected_photons_filter(self, value: int):
```

Comment:
Why have this method in both 'noisy_sampling_simulator.py` script and here?

Reply:
We've got two specialized simulators: one for strong simulation, the other for sampling. They both share as much API as possible. For that reason, the same call exists in both classes.